spec/system/task_spec.rb

```diff
@@ -1,24 +1,26 @@
 require 'rails_helper'
 
 RSpec.describe 'Task', type: :system do
+  let(:project) { create(:project) }
+  let(:task) { create(:task, project_id: project.id) }
```

Comment:
`association`をつけると`project_id`の指定は省略できますね。
```suggestion
  let(:task) { create(:task) }
```

Reply:
efc603b19a86902b1d6c639def9bd2542a35034eで`project_id:  project.id`を削除しました。

Reply:
efc603b19a86902b1d6c639def9bd2542a35034e で`project_id: project.id` を削除しました。